prow/plugins/lifecycle/reopen.go

```diff
@@ -63,37 +63,39 @@ func handleReopen(gc githubClient, log *logrus.Entry, e *github.GenericCommentEv
 		return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, commentAuthor, response))
 	}
 
-	// Add a comment before reopening the PR or issue
-	// to leave an audit trail of who asked to reopen it.
 	if e.IsPR {
-		response := fmt.Sprintf("Reopening this PR.")
-		if err := gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, commentAuthor, response)); err != nil {
-			log.WithError(err).Errorf("Failed adding comment while reopening the PR")
-		}
-
 		log.Infof("/reopen PR")
 		err := gc.ReopenPR(org, repo, number)
 		if err != nil {
 			if scbc, ok := err.(github.StateCannotBeChanged); ok {
 				resp := fmt.Sprintf("failed to re-open PR: %v", scbc)
 				return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
 			}
+		} else {
+			// Add a comment after reopening the PR to leave an audit trail of who
+			// asked to reopen it.
+			response := fmt.Sprintf("Reopened this PR.")
```

Comment:
nit: nothing to format?

Reply:
also should message generally start with upper or lower case? maybe unify with the `failed to re-open` message above?

Reply:
The first word in a sentence should be capitalized. This isn't an `error` type

Reply:
Shouldn't `pull-test-infra-verify-govet` catch the `Sprintf` issue?

Reply:
this is valid sprintf? it's just unnecessary. go vet does not complain about this